src/org/freenetproject/freemail/wot/WoTConnectionImpl.java

```diff
@@ -88,56 +88,25 @@
 	}
 
 	@Override
-	public Set<Identity> getAllTrustedIdentities(String trusterId) {
-		return getAllIdentities(trusterId, TrustSelection.TRUSTED);
-	}
-
-	@Override
-	public Set<Identity> getAllUntrustedIdentities(String trusterId) {
-		return getAllIdentities(trusterId, TrustSelection.UNTRUSTED);
-	}
-
-	private Set<Identity> getAllIdentities(String trusterId, TrustSelection selection) {
-		if(trusterId == null) {
-			throw new NullPointerException("Parameter trusterId must not be null");
-		}
-		if(selection == null) {
-			throw new NullPointerException("Parameter selection must not be null");
-		}
-
+	public Set<Identity> getAllIdentities() {
 		SimpleFieldSet sfs = new SimpleFieldSetFactory().create();
-		sfs.putOverwrite("Message", "GetIdentitiesByScore");
-		sfs.putOverwrite("Truster", trusterId);
-		sfs.putOverwrite("Selection", selection.value);
-		sfs.put("WantTrustValues", false);
-
-		/*
-		 * The Freemail context wasn't added prior to 0.2.1, so for a while after 0.2.1 we need to
-		 * support recipients without the Freemail context.
-		 * FIXME: Restrict this to Freemail context a few months after 0.2.1 has become mandatory
-		 */
-		sfs.putOverwrite("Context", "Freemail");
+		sfs.putOverwrite("Message", "GetIdentities");
+		sfs.putOverwrite("Context", "");
 
 		Message response = sendBlocking(new Message(sfs, null), "Identities");
-		if(response == null) {
-			return null;
-		}
-		if(!"Identities".equals(response.sfs.get("Message"))) {
-			return null;
-		}
+		if(response == null) return null;
+		if(!"Identities".equals(response.sfs.get("Message"))) return null;
```

Comment:
You can create a single `if` with two conditions, though.

Reply:
If that’s too unreadable, extract it into a method with a nice, readable name. :)